.github/workflows/release-create-branch.yml

```diff
@@ -0,0 +1,36 @@
+name: Release - Create Branch
+
+on:
+  workflow_dispatch:
+    inputs:
+      gitRef:
+        description: "Git Ref (e.g. main-456)"
+        required: true
+        default: ""
+      releaseType:
+        description: "Release type"
+        type: choice
+        required: false
```

Comment:
Why is releaseType not a required input?  Am I missing something?

Reply:
No reason, I will change it to true.

Reply:
Changed!